napari/layers/utils/text_manager.py

```diff
@@ -76,6 +76,12 @@ class TextManager(EventedModel):
         Angle of the text elements around the anchor point. Default value is 0.
     """
 
+    class Config:
+        # override EventedModel which defaults to 2 (inplace mutation)
+        # note that if we wanted some fields to have inplace mutations and some not,
+        # we would still have to set this to 1 or the global setting would win
+        allow_mutation = 1
```

Comment:
Could we use named contraints instead of numbers?

Reply:
Yes; we can even use an `extra` attribute, which pydantic supports; it just gets more verbose and tricky to work with, but it's definitely doable.